docs/Limitations.md

```diff
@@ -216,6 +219,27 @@ Equivalent functionality can be achieved with the newer docker networking comman
 See more documentation at
 [docs.docker.com](https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/).
 
+## Storage limitations
+
+### Block devices
+
+Currently, block devices are mounted on the host and passed to the guest VM
```

Comment:
I don't see this as a limitation, but an implementation detail on how bind-mounted volumes are handled. Again, at best its confusing since we do support kubernetes raw block device volumes and we _do_ support standard volumes as well.

Reply:
Hey @egernst, @fidencio reminded me that I had not followed up here.

I would like to understand if your position is that:
* we document as "Limitations" only functional limitations, and not performance limitations (or relatively minor functional impact), and that you object to any documentation about the impact of virtiofs under "limitation",

or

* we should not single out block devices because other volumes have the same prolbme, and that you would rather rephrase the limitation to not single-out block devices?

I still think that the practical impact of virtiofs is high enough that we should probably mention it somewhere.

Reply:
AFAIU, @egernst's point is that this is more an "implementation detail", that happens to hit the users with a performance penalty, but this is not something we should document.

In any case, I'm fine either way, as long as we can get into a consensus between @c3d and @egernst.

If we want to close this one, I'll leave that to @c3d to do so.

Thanks to both of you for the PR & the discussions.

Reply:
I wouldn't put it in limitations. The list is functional issues there; not performance overhead.

If we want to quantify performance overheads and publish, then I'm definitely happy to see that done. We should do similar for network and latency. Its complicated, though. With virtiofs+DAX, you could have near native performance, comparable or better than virtiob-lk...